src/NexusMods.App.UI/Windows/MainWindowViewModel.cs

```diff
@@ -88,32 +96,20 @@ public class MainWindowViewModel : AViewModel<IMainWindowViewModel>
                 })
                 .DisposeWith(d);
 
-            this.WhenAnyValue(vm => vm.Spine.LeftMenu)
-                .Select(left =>
-                {
-                    logger.LogDebug("Spine changed left menu to {LeftMenu}", left);
-                    return left;
-                })
-                .BindTo(this, vm => vm.LeftMenu)
-                .DisposeWith(d);
-
-            this.WhenAnyValue(vm => vm.LeftMenu.RightContent)
-                .Select(right =>
-                {
-                    logger.LogDebug(
-                        "Left menu changed right content to {RightContent}",
-                        right);
-                    return right;
-                }).BindTo(this, vm => vm.RightContent);
-
             // Only show the updater if the metrics opt-in has been shown before, so we don't spam the user.
             if (!metricsOptInViewModel.MaybeShow())
                 updaterViewModel.MaybeShow();
 
+            this.WhenAnyValue(vm => vm.IsActive)
+                .Where(isActive => isActive)
+                .Select(_ => WindowId)
+                .BindTo(_windowManager, manager => manager.ActiveWindowId)
```

Comment:
What is the advantage of this syntax over `.Subscribe(_ => _windowManager.ActiveWindowId = WindowId)` ?

Reply:
See UI coding guidelines, setting values using `Subscribe` should generally be avoided.